tests/post_training/pipelines/base.py

```diff
@@ -212,6 +213,12 @@ def quantize(self) -> None:
         Run quantization of the model and collect time and memory usage information.
         """
         print("Quantization...")
+
+        if self.backend in [BackendType.TORCH, BackendType.OLD_TORCH]:
+            cpu_threads_num = os.environ.get("CPU_THREADS_NUM")
```

Comment:
What do you think about getting number of CPU processes using the following function https://github.com/openvinotoolkit/nncf/blob/ffa15d92032543afeac457376a4db120f8872f6e/nncf/common/utils/os.py#L45?

Reply:
The root cause of slow quantization is limit of cpu usage by configuration of Kubernetes but torch try use all available threads. `get_available_cpu_count` returns the number of cores in the processor despite the config limits.